src/lbaf/Execution/lbsClusteringTransferStrategy.py

```diff
@@ -29,6 +29,12 @@ def __init__(self, criterion, parameters: dict, lgr: Logger):
         self._logger.info(
             f"Enter subclustering immediately after cluster swapping: {self.__separate_subclustering}")
 
+        # Initialize percentage of maximum load required for subclustering
+        self.pct_of_max_load = 0.1
+
+        # Initialize fraction of local imbalance that must be resolved by subcluster
+        self.minimum_fraction_of_local_imbalance = 0.1
+
```

Comment:
Should these be configurable parameters? If so, what would be good default values?

Reply:
Yes, this should be configurable.